thavalon-server/src/game/role.rs

```diff
@@ -44,8 +44,21 @@ pub struct RoleDetails {
     other_info: String,
     /// Special abilities the player has.
     abilities: String,
-    /// Whether or not the player can be assassinated.
+    /// Whether or not the player can be assassinated. Only ever true for good players.
     assassinatable: bool,
+    /// Whether or not the player is the Assassin. Only ever true for evil players.
+    is_assassin: bool,
+    /// If the player is the Assassin, this is the Priority Target that they may assassinate.
+    priority_target: Option<PriorityTarget>,
+}
+
+/// A priority assassination target. If the Good team passes 3 missions, then the Assassin must correctly identify
+/// the Priority Target in order to win.
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub enum PriorityTarget {
+    Merlin,
+    Lovers,
+    Guinevere,
 }
```

Comment:
This isn't quite right with  the version of assassination we want, but we'll still need a separate enum from `Role` to handle lovers